src/CSAccounting.sol

```diff
@@ -300,6 +315,45 @@ contract CSAccounting is
         MODULE.updateDepositableValidatorsCount(nodeOperatorId);
     }
 
+    /// @inheritdoc ICSAccounting
+    function increaseBondReserve(
+        uint256 nodeOperatorId,
+        uint256 newAmount
+    ) external whenResumed whenBondReserveIsEnabled {
+        _checkAndGetEligibleNodeOperatorProperties(nodeOperatorId);
+
+        if (newAmount == 0) {
+            revert InvalidBondReserveAmount();
+        }
+
+        (uint256 current, uint256 required) = getBondSummary(nodeOperatorId);
+        uint256 reserve = BondReserve.getReservedBond(nodeOperatorId);
+        // NOTE: `required` is always >= `reserve` due to fact that `reserve` is a part of `required`
+        if (reserve > newAmount || current < (required - reserve) + newAmount) {
+            revert InvalidBondReserveAmount();
+        }
+        BondReserve._setBondReserve(nodeOperatorId, newAmount);
+    }
+
+    /// @inheritdoc ICSAccounting
+    function removeBondReserve(
```

Comment:
removing bond reserve can affect depositable vals count (unbonded -> depositable) so I believe you need to call updateDepositable at the end